Author: Daniel Baumann <daniel@debian.org>
Description:
 Parts of the locales can not be customized through the &brandShortName; and
 &brandFullName; entities, hence we have to replace Thunderbird with Icedove
 manually.

diff -Naur en-GB.orig/chrome/locale/en-GB/messenger/aboutDialog.dtd en-GB/chrome/locale/en-GB/messenger/aboutDialog.dtd
--- en-GB.orig/chrome/locale/en-GB/messenger/aboutDialog.dtd	2007-01-30 03:15:52.000000000 +0000
+++ en-GB/chrome/locale/en-GB/messenger/aboutDialog.dtd	2008-02-02 16:55:24.000000000 +0000
@@ -2,6 +2,4 @@
 <!ENTITY credits                "Credits">
 <!ENTITY aboutLink              "&lt; About &brandFullName;">
 <!ENTITY aboutVersion           "version">
-<!ENTITY copyrightText          "&#169;1998-2007 Contributors. All Rights Reserved. Mozilla Thunderbird and the
-                                 Thunderbird logos are trademarks of the Mozilla Foundation.  All rights 
-                                 reserved.">
+<!ENTITY copyrightText          "&#169;1998-2007 Contributors. All Rights Reserved.">
